### PR TITLE
Bugfix of htconfig.php and friendica script

### PR DESCRIPTION
**Pull request by nupplaphil**
Fixed a `.bin/friendica` copy bug
Fixed the `.config/htconfig.php` for MariaDB use

### CODE REVIEW

**Comment by nupplaphil** [comment]
The `.config/htconfig.php` is based on https://github.com/friendica/friendica/pull/5131

File: .config/htconfig.php [hunk 1]
```diff
@@ -6,11 +6,15 @@
 // Use environment variables for mysql if they are set beforehand
 if (!empty(getenv('MYSQL_HOST'))
 	&& !empty(getenv('MYSQL_PORT'))
-	&& !empty(getenv('MYSQL_USERNAME'))
+	&& (!empty(getenv('MYSQL_USERNAME')) || !empty(getenv('MYSQL_USER')))
```
Comment (MrPetovan) [inline .config/htconfig.php]: Why are both versions cohabiting? Can't we just use one of them?

Comment (nupplaphil) [reply .config/htconfig.php]: Mariadb => MYSQL_USER
MySQL => MYSQL_USERNAME

I want to be open for mysql and mariadb installations :-)

Comment (MrPetovan) [reply .config/htconfig.php]: Shaking my head.

Comment (nupplaphil) [reply .config/htconfig.php]: mine too, took me a while to find it ;-)

**Pull request merged**